Swerve2022/src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -38,6 +38,8 @@ public class Drivetrain extends SubsystemBase
   private final Translation2d         m_backLeftLocation   = new Translation2d(-0.381, 0.381);
   private final Translation2d         m_backRightLocation  = new Translation2d(-0.381, -0.381);
 
+  private final String                kcanbusstring        = DTConsts.kcanbusstring;

```

Comment:
Not sure why we need a member variable to hold this constant value. Just use the constant wherever needed.